RealDeviceMap-UIControl/Misc.swift

```diff
@@ -160,7 +160,7 @@ extension XCTestCase {
     }
 
     func checkHasWarning(screenshot: XCUIScreenshot?=nil) -> Bool {
-        Log.debug("Checking for the warning pop-up")
+        Log.debug("Checking for the red warning")
```

Comment:
with us reading "GetPlayerResponse" do we even need that check any more?

Reply:
Gpr often misses rw. If you don't think it's needed we should remove the check entirely.

Reply:
That needs to be discussed.
Haven't looked into if how warnings work in GPRs